pkg/api/routes_test.go

```diff
@@ -46,9 +46,10 @@ func TestRoutes(t *testing.T) {
 			panic(err)
 		}
 
-		go startServer(ctlr)
-		defer stopServer(ctlr)
-		test.WaitTillServerReady(baseURL)
+		cm := test.NewControllerManager(ctlr)
+		cm.StartServer()
```

Comment:
StartAndWait(port) <- one call? this is usually the call everyone makes

Reply:
Add this as another API btw, and call that instead here, we may need a separate StartServer() call elsewhere